tests/app/service/test_discovery.py

```diff
@@ -50,6 +50,7 @@
     ],
     "framework": {"name": "langchain", "version": "0.1.45"},
     "plugin_version": "0.1.1",
+    "client_version": {"name": "langchain_community_version", "version": "x.x.x"},
```

Comment:
nit: `name` should be just pkg name i.e. `langchain_community`
```suggestion
    "client_version": {"name": "langchain_community", "version": "x.x.x"},
```